2021q3/azure.adoc

```diff
@@ -0,0 +1,35 @@
+=== FreeBSD on Microsoft HyperV and Azure
+
+Links: +
+link:https://wiki.freebsd.org/MicrosoftAzure[Microsoft Azure article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/MicrosoftAzure[] +
+link:https://wiki.freebsd.org/HyperV[Microsoft HyperV article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/HyperV[]
+
+Contact: Microsoft FreeBSD Integration Services Team <bsdic@microsoft.com> +
+Contact: link:https://lists.FreeBSD.org/mailman/listinfo/freebsd-cloud[freebsd-cloud Mailing List] +
+Contact: The FreeBSD Azure Release Engineering Team <releng-azure@FreeBSD.org> +
+Contact: Wei Hu <whu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+
+The link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-13_0[13.0-RELEASE image on Azure Marketplace] has been link:https://lists.freebsd.org/pipermail/freebsd-announce/2021-September/002059.html[published].
+
+The changes for building official images on Azure Marketplace, which fulfill the requirements of Azure and FreeBSD cloud images like disk layout and UEFI for Gen2 VM, along with some minor improvements like configurations to speed up booting, has been link:https://cgit.freebsd.org/src/commit/?id=fbece7609573bd51080e949df03fa3d803a637ae[imported].
+
+Above tasks are sponsored by The FreeBSD Foundation, with resources provided by Microsoft.
+
+Microsoft Azure Network Adapter (MANA) is the new network adapter from Microsoft which will be available in Azure public cloud.
+It provides SRIOV NIC as virtual function to guest OS running on Hyper-V.
```

Comment:
s/as virtual function to guest/as a virtual function to a guest/ (also, should "function" be "driver" here?